### PR TITLE
prefill search bar

### PR DESCRIPTION
**Pull request by sakshee-19**
**Checklist**

- [X] My branch is up-to-date with the upstream `develop` branch.
- [ ] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: 
fixes: #21 

Adds functionality to auto fill search bar before hand with random topics from metadata.json to help new users in running project more easily.

**Why do we need this PR?**:
Random filled topic:
screenshot from 2018-02-11 11-03-32

Random filled topic:
screenshot from 2018-02-11 11-03-39

### CODE REVIEW

**Comment by sakshee-19** [comment]
@AdiChat Please review.

**Comment by ankan17** [comment]
@sakshee-19 It is general convention while contributing to an open-source project that we first have to comment on the issue on which we are working so that other developers can know that someone else is working on it and they don't waste their time. 😅

**Comment by sakshee-19** [comment]
@ankan17 There was no response from you for last 2 days. It is supposed to update status of work on issue you are working. It is more conventional to follow one issue one PR rule. I am new to open source and will keep it in consideration surely from next time. Apologies for the trouble. Thanks.

**Comment by AdiChat** [comment]
Nice work 👍 

There are a couple of issues like it may suggest `src` and `test` as well. Instead of fetching the suggested term from the metadata, you may get hold of a random term from `tags.json`. See the structure of the file [here](https://github.com/OpenGenus/cosmos-search/pull/43/files#diff-b61a64c5a829f11c30748603d2e4c8d0). You may take this approach as the work on the list of tags is going on parallelly. 👍 

There seems to have been a minor confusion while contributing to this feature. No worries 😄 We shall keep this in mind as we move forward. 👍

**Comment by sakshee-19** [comment]
@AdiChat so, can I continue my work? should I use ```tags.json``` assuming work on it is completed or try to resolve suggested issues by modifying my work?

**Comment by AdiChat** [comment]
You may assume `tags.json` is ready and continue to deliver this feature though we may be able to check it once the concerned work is completed. 👍

**Comment by sakshee-19** [comment]
As soon as the ```tag.json``` file will be available, I will make the PR for this issue.

**Comment by ankan17** [comment]
@sakshee-19 You can take reference from my PR #43. The `tags.json` file will have the same structure as of now. You can modify your code accordingly to accommodate the changes :)

**Comment by sakshee-19** [comment]
@ankan17  Whether the ```tag.json``` file will keep the algo-tags that contains files atleast one code file other than ```README.md``` or it will keep all the names.

**Comment by sakshee-19** [comment]
@AdiChat Should I include all the algo titles present in ```tags.json```. There are some algo titles in ```tag.json``` for which there is no code available in the repository. So, shall I check it with ```metadata.json``` to filter out those algo titles which are creating this issue ?

File: search/views.py [hunk 1]
```diff
@@ -4,10 +4,22 @@
...
 
-
 # Create your views here.
+
+# To prefill the searchbar
+def searchbar():
+    jsonFile = open('tags.json', 'r')
+    data = json.load(jsonFile)
+    algo_list = data['tags']
+    print(algo_list[0])
```
Comment (vaibhavsingh97) [inline search/views.py]: Please remove `print` statement

**Comment by sakshee-19** [comment]
@AdiChat Please review. It will require ```tag.json``` file. I have tested it locally with temporary ```tag.json``` file but did not include it.

**Comment by ankan17** [comment]
@sakshee-19 It'd be better if you could merge all those commits as the other two commits are basically cleanup commits :) Maintainers will know better :) @AdiChat @vaibhavsingh97

**Comment by AdiChat** [comment]
@sakshee-19 Could you share the tag that may return an internal error? We must explore the cause. 

I observed that tags with `-` originally are returning no results due to the search algorithm used. 
We need to make suitable adjustments as we do not want the suggestions to fail.

**Comment by sakshee-19** [comment]
@AdiChat like ```factorization machines```, actually its not the error but a page showing ```could not find the results```.

**Comment by AdiChat** [comment]
I see. We need to improve our search strategy. 👍 
Anyways, this feature has been developed perfectly.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**